packages/dds/shared-object-base/src/test/serializer.spec.ts

```diff
@@ -5,8 +5,6 @@
 
 import { strict as assert } from "assert";
 
-import { IFluidHandle } from "@fluidframework/core-interfaces";
```

Comment:
I guess this callback didn't really have test coverage - oh well, it seemed to work :)